cli/generator-op-microservice/generators/microservice/templates/rest.service.ts

```diff
@@ -14,8 +14,8 @@ import mongoose from 'mongoose';
 import <%= apiName %>Router from './src/api';
 
 const httpsOptions = {
-  key: fs.readFileSync('./certs/server.key'),
-  cert: fs.readFileSync('./certs/server.crt')
+  key: fs.readFileSync(`${ process.env.SSL_KEY }`),
+  cert: fs.readFileSync(`${process.env.SSL_CERT }`)
```

Comment:
```suggestion
  key: fs.readFileSync(process.env.SSL_KEY),
  cert: fs.readFileSync(process.env.SSL_CERT)
```

Reply:
@deshmukhmayur that approac thows the following error

`Argument of type 'string | undefined' is not assignable to parameter of type 'string`